### DISCUSSION TITLE
Locational Marginal Cost

### DISCUSSION THREAD

**Discussion by Crueda96** [Critical-Infrastructure-Systems-Lab/PowNet#12] [Q&A]
Dear Sir,
Thank you for your efforts.

Is it possible to calculate the nodal LMP prices for each bus considering the transmission losses?

**Comment by kamal0013** [comment]
Hi Christian,

Thanks for asking but I think it is not possible to estimate the cost at each node. The model mainly estimates the system-wide cost.

Best,
Kamal
<https://www.linkedin.com/in/kamalchowdhury13>
________________________________
From: Christian Rueda ***@***.***>
Sent: Sunday, July 11, 2021 5:19 AM
To: kamal0013/PowNet ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [kamal0013/PowNet] Locational Marginal Cost (#12)


Dear Sir,
Thank you for your efforts.

Is it possible to calculate the nodal LMP prices for each bus considering the transmission losses?
